zzk/service/mocks/RegisteredHostHandler.go

```diff
@@ -22,19 +22,19 @@ type RegisteredHostHandler struct {
 	mock.Mock
 }
 
-func (_m *RegisteredHostHandler) GetRegisteredHosts(cancel <-chan interface{}) ([]host.Host, error) {
-	ret := _m.Called(cancel)
+func (_m *RegisteredHostHandler) GetRegisteredHosts(pool string) ([]host.Host, error) {
```

Comment:
ditto my earlier concern about dropping the `cancel` argument.  I'm not familiar with how this is all _supposed_ to work, so maybe it's OK.